### PR TITLE
gdrive: add page

### PR DESCRIPTION
**Pull request by Jonathanfk1**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/gdrive.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# gdrive
+
+> Terminal tool for Google Drive operations. Folder/file id can be obtained from the Google Drive folder or id url.
```
Comment (pxgamer) [inline pages/linux/gdrive.md]: Perhaps break this into 2 lines. So:

> Terminal tool for Google Drive operations.
> Folder/file id can be obtained from the Google Drive folder or id url.

File: pages/linux/gdrive.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Upload local 'path' to -p 'id' parent folder:
```
Comment (pxgamer) [inline pages/linux/gdrive.md]: Not sure if `-p` is really required here. Or the quote marks around `path` and `id`. Perhaps something like this? (If this is what it does)

> Upload a local path to the parent folder with the specified id:

File: pages/linux/gdrive.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`gdrive upload -p {{id}} {{path}}`
+
+- Download file or directory by 'id' to current path:
+
+`gdrive download {{id}}`
+
+- Download file or directory by id to local --p 'path':
```
Comment (pxgamer) [inline pages/linux/gdrive.md]: Same again here. Maybe:

> Download a file or directory by its id to a local path:

**Comment by Jonathanfk1** [comment]
yeah @pxgamer, your suggestions make sense, thanks! :smile:

File: pages/linux/gdrive.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`gdrive upload -p {{id}} {{path}}`
+
+- Download file or directory by id to current path:
+
+`gdrive download {{id}}`
+
+- Download a file or directory by its id to a local path:
+
+`gdrive download --p {{path}} {{id}}`
```
Comment (MasterOdin) [inline pages/linux/gdrive.md]: Looking at the [readme for download](https://github.com/prasmussen/gdrive/blob/master/README.md#download-all-files-and-directories-matching-query), shouldn't this be `--path`?

**Comment by MasterOdin** [comment]
This page should probably be under common assuming it's referencing/using https://github.com/prasmussen/gdrive?

File: pages/linux/gdrive.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# gdrive
+
+> Terminal tool for Google Drive operations.
```
Comment (agnivade) [inline pages/linux/gdrive.md]: How about - "Command line tool to interact with Google Drive" ?

File: pages/linux/gdrive.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Download file or directory by id to current path:
+
+`gdrive download {{id}}`
+
+- Download a file or directory by its id to a local path:
+
+`gdrive download --p {{path}} {{id}}`
+
+- Create revision of id file using file specified by path:
```
Comment (agnivade) [inline pages/linux/gdrive.md]: "Create a new revision of an id using a given file."

We have been mentioning "file or directory by its id" so far, so it is implicit by now.

**Comment by agnivade** [comment]
@Jonathanfk1 - If the tool is what @MasterOdin mentions, we should move it to common.

**Comment by Jonathanfk1** [comment]
Thank you guys for the suggestions. How do I change the file directory to this PR? I've changed it locally but I don't know how to attach my commit to this PR.

File: pages/linux/gdrive.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`gdrive upload -p {{id}} {{path/to/file_or_folder}}`
+
+- Download file or directory by id to current directory:
+
+`gdrive download {{id}}`
+
+- Download to a given local path by its id to a given local path:
+
+`gdrive download --path {{path/to/file_or_folder}} {{id}}`
```
Comment (agnivade) [inline pages/linux/gdrive.md]: shouldn't this be path/to/folder ? I don't think you can download it to a file.

File: pages/linux/gdrive.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a new revision of an id using a given file:
```
Comment (agnivade) [inline pages/linux/gdrive.md]: "given file or folder" - to match with the example

**Comment by agnivade** [comment]
> How do I change the file directory to this PR?

Just do a `git mv <old_path> <new_path>` and commit your changes and push.

File: pages/linux/gdrive.md (hunk 8 above)
Comment (agnivade) [reply pages/linux/gdrive.md]: Sorry, I would have done this change myself, but you don't seem to have given me edit permissions. Kindly take a look at this. :)

**Comment by pxgamer** [comment]
@agnivade, is this good to go?

**Pull request merged**